arviz/plots/traceplot.py

```diff
@@ -25,6 +25,7 @@ def plot_trace(
     rug_kwargs=None,
     hist_kwargs=None,
     trace_kwargs=None,
+    max_plots=40,
```

Comment:
I have set a preliminary maximum here, which will be removed when we decide on the implementation of global variables #609

Reply:
We should have a warning, telling users that want to plot more than `max_plot` plots that only `max_plots` are actually plotted.

Reply:
Maybe `max_plot` should also accepts `None`, in case users does not want to put a limit

Reply:
I agree on the warning, I will add it as soon as possible. About accepting `None`, I think it is not necessary, if no limit is wanted, it is easy to set it to 10000 and let matplotlib hanging be the limit.

Reply:
I think it may be better to merge this once it the warging is added, and afterwards do another PR to implement package parameters.

Reply:
Right about setting something to a large number but is more elegant to have a special keyword for that case. Anyway, I agree we should add the warning and then merge.

Reply:
I added both, `None` option and warning